github/actions/config.go

```diff
@@ -102,8 +102,7 @@ func isHostedGitHubURL(u *url.URL) bool {
 		return false
 	}
 
-	return strings.EqualFold(u.Host, "github.com") ||
-		strings.EqualFold(u.Host, "www.github.com") ||
+	return strings.HasSuffix(u.Host, "github.com") ||
```

Comment:
if we change this, does it means we also need to change the runner?

Reply:
I think this should be:
```go
strings.EqualFold(u.Host, "github.com") || strings.HasSuffix(u.Host, ".github.com")
```
Domain "examplegithub.com" would pass this check if we just check for the suffix

Reply:
Good catch @nikola-jokic!

Reply:
@TingluoHuang I'm not familiar how this impacts the runner, can you or @nikola-jokic elaborate on the impact please?